src/wallet/subcommands/wallet_cmd.rs

```diff
@@ -204,7 +221,7 @@ pub enum WalletCommands {
     /// Get account balance
     Balance {
         /// The address of the account to check
-        address: String,
+        address: Address,
```

Comment:
This is a regression. Using `Address` rather than `String + StrictAddress` silently ignores mainnet/testnet address mismatches. For the safe handling of funds, it's important to fail if the user, for example, tries to send FIL from a mainnet address to a testnet address.

Reply:
I'm not sure how to write a unit test to catch this.

Reply:
Could we have a test in https://github.com/ChainSafe/forest/blob/main/scripts/tests/calibnet_wallet_check.sh for this?

Reply:
I'll punt the address test to https://github.com/ChainSafe/forest/issues/2945